mailpoet/assets/css/src/components-automation-listing/_sections.scss

```diff
@@ -0,0 +1,171 @@
+:root {
+  --menu-width: 0;
+
+  @media screen and (min-width: 783px) {
+    --menu-width: 36px;
+  }
+
+  @media screen and (min-width: 961px) {
+    --menu-width: 160px;
+  }
+}
+
+@mixin full-width {
+  margin-left: calc(-50vw + var(--menu-width) / 2 + 50%);
+  padding-left: 104px;
+  padding-right: 104px;
+  width: calc(100vw - var(--menu-width));
```

Comment:
You seem to be setting `100vh` and then subtracting the (expected) menu, is that correct? Is there a reason for that? It seems to be unnecessarily complex to me, and replicating things like `min-width: 783px` and `--menu-width: 36px` is not nice either. I think this may also break if some plugins change the admin styles, etc.

So now the "why don't you just" question 😄 Why don't you just fill the whole content part with sth like `width: 100%`?

Reply:
bad67451a

Reply:
After rebase: [19b5349](https://github.com/mailpoet/mailpoet/pull/4461/commits/19b5349a9808bebf5124fa7774152fb8c4c4cabf)